framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -3130,6 +3154,26 @@ protected void checkThrownExpression(ThrowTree tree) {
         }
     }
 
+    /**
+     * Returns a set of AnnotatedTypeMirror that is allowed throw exceptions for method.
+     *
+     * <p>Note: If the method declaration do not have any throw clauses, then return null.
+     *
+     * @param mtree MethodTree to look up throw expressions
+     * @return List of AnnotationTypeMirrors, one per exception in method signature, that contains
+     *     annotation and java expressions
+     */
+    protected List<AnnotatedTypeMirror> getExceptionList(MethodTree mtree) {
```

Comment:
Why isn't https://github.com/eisop/checker-framework/blob/bf039c38d3bfe8fe5e567456fb6927851f225177/framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeMirror.java#L1509 returning this information already? Is there a difference in the returned types?

Reply:
> Why isn't
> 
> https://github.com/eisop/checker-framework/blob/bf039c38d3bfe8fe5e567456fb6927851f225177/framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeMirror.java#L1509
> 
> returning this information already? Is there a difference in the returned types?

Thanks. I tried and it works. So I will remove the method and `getThrownTypes`. Also resolve related comments.